guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,28 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
+
+. Create a backup or take a snapshot.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+
+. In the {ProjectWebUI} , navigate to *Monitor* > *Jobs*.
+
+. Click *Run Job*.
+
+. From the *Job category* list, select *Maintenance Operations*.
+
+. From the *Job template* list, select *{SmartProxy} Upgrade Playbook*.
+
+. In the *Search Query* field, enter the host name of the {SmartProxy}.
+
+. Ensure that *Resolves to* field displays *1 host*.
+
+. In the *target_version* field, enter the target version of the {SmartProxy}.
```

Comment:
Now I am wondering whether we should submit bugs for these "name_with_underscore" field labels. They look like variables that ended up as labels.

Reply:
@apinnick should I mark this as resolved for now?